### PR TITLE
fix(iOS): ListView empty items when scrolling

### LINKED ISSUE #9416: [iOS] List items are missing on fast scroll
### Current behavior

Sometimes when the samples list in the iOS Samples app is scrolled fast, some of the items are blank. Only when scrolled away and back in slowly they appear. The empty space is not interactive, so it is no longer possible to tap on that item until it actually appears.

MicrosoftTeams-image (4)
MicrosoftTeams-image (5)

This behavior has been present for a longer time, so not connected to recent changes.

### Expected behavior

Should appear on first try.

### How to reproduce it (as minimally and precisely as possible)

_No response_

### Workaround

_No response_

### Works on UWP/WinUI

Yes

### Environment

_No response_

### NuGet package version(s)

_No response_

### Affected platforms

iOS

### IDE

_No response_

### IDE version

_No response_

### Relevant plugins

_No response_

### Anything else we need to know?

_No response_

### PR DESCRIPTION
**Pull request by ajpinedam**
GitHub Issue (If applicable): closes #9416

## PR Type

- Bugfix

## What is the current behavior?
When Scrolling over a ListView some items are not properly measured (not visible because of their size)

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by ajpinedam** [comment]
## Before
https://user-images.githubusercontent.com/1900897/188788139-0b88b9ba-a357-48af-a0dc-03f88eb1ff81.mp4


## After
https://user-images.githubusercontent.com/1900897/188788152-2c7a7ab2-3e97-44f5-8ffe-c2942a29c7b6.mp4

**Comment by MartinZikmund** [comment]
Magic! 🪄

**Comment by MartinZikmund** [comment]
@ajpinedam Please check these runtime tests: 
image

Seems they might be caused by the change, not sure how

File: src/Uno.UI/UI/Xaml/FrameworkElement.iOS.cs [hunk 1]
```diff
@@ -64,8 +64,6 @@ public override void LayoutSubviews()
 						var parent = Superview;
 						if (parent is UIElement
 						    || parent is ISetLayoutSlots
-						    // In the case of ListViewItem inside native list, its parent's parent is ListViewBaseInternalContainer
-						    || parent?.Superview is ISetLayoutSlots
```
Comment (jeromelaban) [inline src/Uno.UI/UI/Xaml/FrameworkElement.iOS.cs:68]: Very nice! Let's find out why this was originally needed.

Comment (spouliot) [reply src/Uno.UI/UI/Xaml/FrameworkElement.iOS.cs:68]: definitively not what I expected 😄

Comment (jeromelaban) [reply src/Uno.UI/UI/Xaml/FrameworkElement.iOS.cs:68]: @ajpinedam this was added in https://github.com/unoplatform/uno/pull/7399/files#diff-24c92c6bd56b6e501b90b4d9ccc6a72ea89c32b20b2992241cc7f072c0c42ddb. Would you know if the `When_ItemsPresenter_MinHeight` test is passing?

File: src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBaseSource.iOS.cs [hunk 2]
```diff
@@ -743,6 +743,14 @@ public ContentControl Content
 
 					ContentView.AddSubview(value);
 
+
+					Layouter.ArrangeChild(value, new Rect(0, 0, (float)Frame.Width, (float)Frame.Height));
```
Comment (ajpinedam) [inline src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBaseSource.iOS.cs:747]: @jeromelaban @dr1rrb  will doing this generate a lot of "noise"? This, combined with the `UpdateContentLayoutSlots()` works perfect for all the unit tests

Comment (dr1rrb) [reply src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBaseSource.iOS.cs:747]: Humm, it's weird that we arrange child/content without even measuring it. I'm worried that we could arrange twice each item in the list. Does the `ClearMeasuredSize` below cause a layout pass?

Also looking at your comment, is it possible that we are bypassing that internal container during layouting ?

**Review by spouliot — APPROVED** [review]

**Review by MartinZikmund — APPROVED** [review]

**Comment by MartinZikmund** [comment]
@Mergifyio backport release/stable/3.11

**Pull request merged**